Lib/test/test_generated_cases.py

```diff
@@ -412,10 +412,12 @@ def test_predictions(self):
             frame->instr_ptr = next_instr;
             next_instr += 1;
             INSTRUCTION_STATS(OP1);
-            PREDICTED(OP1);
-            _PyStackRef res;
-            res = Py_None;
-            stack_pointer[-1] = res;
+            PRED_OP1:
```

Comment:
```suggestion
            PREDICTED_OP1:
```

`PRED` could be short for "predicate" or "predecessor"